plugin/src/main/scala/chisel3/internal/plugin/ChiselComponent.scala

```diff
@@ -82,9 +82,18 @@ class ChiselComponent(val global: Global, arguments: ChiselPluginArguments)
     // Checking for all chisel3.internal.NamedComponents, but since it is internal, we instead have
     // to match the public subtypes
     private val shouldMatchNamedComp: Type => Boolean =
-      shouldMatchGen(tq"chisel3.Data", tq"chisel3.MemBase[_]", tq"chisel3.VerificationStatement")
+      shouldMatchGen(
+        tq"chisel3.Data",
+        tq"chisel3.MemBase[_]",
+        tq"chisel3.VerificationStatement",
```

Comment:
this all seems good, but does it make sense to make `Data` and `MemBase` etc just also `AffectsChiselPrefix`?